events/event.go

```diff
@@ -48,7 +48,7 @@ type APIVersion string
 
 const (
 	// CurrentAPIVersion specify the lastest API version
-	CurrentAPIVersion APIVersion = "1.0.0"
+	CurrentAPIVersion APIVersion = "0.1.0"
```

Comment:
```suggestion
	CurrentAPIVersion APIVersion = "1.1.0"
```
We don't need to go back in versions. I know that we are the ones primarily consuming this, but I would prefer us to just move forward.

Reply:
@graywolf336 My intent was to start using the version (tag) we generate for each new release. So, I moved from 0.0.3 to 0.1.0.

But what do you think? 🤔 We could keep them separated if you want

Reply:
I think we move forward with the code. I need to bump the version tags.